Added support for rails 5

kopylovvlad opened the pull request.

kopylovvlad committed:
update to rails 5

kopylovvlad committed:
Merge pull request #1 from kopylovvlad/development

update to rails 5

kopylovvlad commented:
Is anybody here? 🤔

kopylovvlad committed:
Update README.rdoc

kopylovvlad committed:
Update README.rdoc

BBenni commented:
I can confirm that this PR ist also working with Rails 6.
Would be nice if it could be accepted.

(Although I am afraid the original maintainer of this gem is not really involved anymore...)

BBenni committed:
Solve problem with current rmagick version 4.2.2

It seems that easy_captcha is not working properly with the current version of the rmagick gem 4.2.2.
The following exception is thrown:
```
 Exception: NoMethodError: undefined method `background_color=' for #<EasyCaptcha::Generator::Default:0x0000564a23cea4f0>
Did you mean?  background_image=
```

The proposed change should be able to solve this problem.

kopylovvlad committed:
Merge pull request #2 from BBenni/patch-1

Solve problem with current rmagick version 4.2.2
